src/tv2_afvd_showstyle/migrations/util.ts

```diff
@@ -105,3 +107,98 @@ export function getOutputLayerDefaultsMigrationSteps(versionStr: string): Migrat
 		})
 	)
 }
+
+/**
+ * "Renames" the id of a table by copying over all values over into a new table which has the new id - then removes the values from the old table
+ */
+export function renameTableId(version: string, oldTableId: string, newTableId: string): MigrationStepShowStyle {
+	return {
+		id: `${version}.rename.table.id.${oldTableId}.to.${newTableId}`,
+		version,
+		canBeRunAutomatically: true,
+		validate: (context: MigrationContextShowStyle) => {
+			const oldConfigTable = (context.getBaseConfig(oldTableId) as unknown) as TableConfigItemValue
+			if (!oldConfigTable || oldConfigTable.length === 0) {
+				return false
+			}
+
+			const newConfigTable = (context.getBaseConfig(newTableId) as unknown) as TableConfigItemValue
+			return !newConfigTable
+		},
+		migrate: (context: MigrationContextShowStyle) => {
+			const oldConfigTable = (context.getBaseConfig(oldTableId) as unknown) as TableConfigItemValue
+			const newConfigTable = ((context.getBaseConfig(newTableId) as unknown) as TableConfigItemValue) ?? []
+			oldConfigTable.map(value => newConfigTable.push(value))
+
+			context.setBaseConfig(newTableId, newConfigTable)
+			context.setBaseConfig(oldTableId, [])
```

Comment:
Why keep the old table instead of removing it?

Reply:
I think this was a combination of looking at how similar places has done it and not knowing the full extent of what is availabe in the "migration api".

Will update

Reply:
Updated